### PR TITLE
[pose-detection]Update BlazePose OneEuro filter config.

### PR DESCRIPTION
**Pull request by lina128**
This PR has below changes:
1. Update OneEuro filter config for BlazePose.
2. Update getObjectScale to use auxiliary bounding box height and width.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/740)

### CODE REVIEW

File: pose-detection/src/blazepose_mediapipe/mediapipe_test.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ const MEDIAPIPE_MODEL_CONFIG: BlazePoseMediaPipeModelConfig = {
 };
 
 const EPSILON_IMAGE = 10;
-const EPSILON_VIDEO = 18;
+const EPSILON_VIDEO = 42;
```
Comment (ivan-grishchenko) [inline pose-detection/src/blazepose_mediapipe/mediapipe_test.ts:33]: Seems like a pretty big diff taking into account that you've updated expected result as well, no?

File: pose-detection/src/blazepose_tfjs/blazepose_test.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ import * as poseDetection from '../index';
 import {getXYPerFrame, KARMA_SERVER, loadImage, loadVideo} from '../test_util';
 
 const EPSILON_IMAGE = 10;
-const EPSILON_VIDEO = 15;
+const EPSILON_VIDEO = 18;
```
Comment (ivan-grishchenko) [inline pose-detection/src/blazepose_tfjs/blazepose_test.ts:28]: Same here

**Review by ivan-grishchenko — APPROVED** [review]

**Review by lina128 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/740)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, and @pyu10055)

___
*[pose-detection/src/blazepose_mediapipe/mediapipe_test.ts, line 33 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/740#-MaB--ld8DWW-VJm8wWW-r1-33:-MaBUCsd5gCOKjwON-zC:b75fv0i) ([raw file](https://github.com/tensorflow/tfjs-models/blob/2f552e1374b119412402386f2d7724bc13ec3317/pose-detection/src/blazepose_mediapipe/mediapipe_test.ts#L33)):*
<details><summary><i>Previously, ivan-grishchenko wrote…</i></summary><blockquote>

Seems like a pretty big diff taking into account that you've updated expected result as well, no?
</blockquote></details>

I think MP Web Solution hasn't updated with the new change, that's why.
___
*[pose-detection/src/blazepose_tfjs/blazepose_test.ts, line 28 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/740#-MaB--ld8DWW-VJm8wWX-r1-28:-MaBUKkM6e5srUlIhWQ-:bcv3pw8) ([raw file](https://github.com/tensorflow/tfjs-models/blob/2f552e1374b119412402386f2d7724bc13ec3317/pose-detection/src/blazepose_tfjs/blazepose_test.ts#L28)):*
<details><summary><i>Previously, ivan-grishchenko wrote…</i></summary><blockquote>

Same here
</blockquote></details>

Yeah, there's a little bit of regression, 3 pixels.

**Review by lina128 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/740)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, and @pyu10055)

___
*[pose-detection/src/blazepose_mediapipe/mediapipe_test.ts, line 33 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/740#-MaB--ld8DWW-VJm8wWW-r1-33:-MaBXuTADTHG6SEj58d7:b-hukn99) ([raw file](https://github.com/tensorflow/tfjs-models/blob/2f552e1374b119412402386f2d7724bc13ec3317/pose-detection/src/blazepose_mediapipe/mediapipe_test.ts#L33)):*
<details><summary><i>Previously, lina128 (Na Li) wrote…</i></summary><blockquote>

I think MP Web Solution hasn't updated with the new change, that's why.
</blockquote></details>

Added TODO.

**Pull request merged**